client/main.js

```diff
@@ -0,0 +1,470 @@
+import BN from "bn.js";
+import web3Utils from "web3-utils";
+
+const ganache = require("ganache-core");
+const contract = require("truffle-contract");
+const ReputationMiningCycleJSON = require("../build/contracts/ReputationMiningCycle.json");
```

Comment:
We could use the loaders / adapters here that we built for the colony-js client to make contract loading more flexible.